app/repositories/application/GeneralApplicationRepository.scala

```diff
@@ -998,6 +1007,74 @@ class GeneralApplicationMongoRepository(timeZoneService: TimeZoneService)(implic
     collection.update(query, passMarkEvaluation, upsert = false) map { _ => }
   }
 
+  // TODO: get rid of this, it feels wrong and we should also be using the App Status case objects.
+  private def applicationStatus(status: String): BSONDocument = {
```

Comment:
Can i have some backstory, where did this appear from? It uses all the old defunct app statuses, so should be good to comment or remove